codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        enabled: yes
+        target: auto
+    patch:
+      default:
+        enabled: yes
+        target: 100%
```

Comment:
@rlucioni Still green build. Either I've been tired and missed something stupid and obvious or it just ignores the codecov.yml for some reason.

I've tried both yaml and yml, if that makes any difference (but again I'm not 100% sure I haven't made a mistake somewhere).